tests/zfs-tests/tests/functional/fault/auto_online_001_pos.ksh

```diff
@@ -54,7 +55,14 @@ fi
 
 function cleanup
 {
-	destroy_pool $TESTPOOL
+	typeset disk
+
+	# Replace any disk that may have been removed at failure time.
+	for disk in $DISK1 $DISK2 $DISK3; do
+		is_real_device $disk && continue
+		insert_disk $disk $(get_scsi_host $disk)
+	done
+	poolexists $TESTPOOL && destroy_pool $TESTPOOL
```

Comment:
You don't need `poolexists $TESTPOOL` because `destroy_pool` already checks if the pool exists.

Reply:
I removed the poolexists call; I only added it to avoid the message in the logs.
I also added a line to skip loop devices, as I think that introduced other errors.